integration-tests/src/test/java/oracle/kubernetes/operator/utils/Domain.java

```diff
@@ -145,6 +145,30 @@ public void verifyDomainCreated() throws Exception {
       callWebAppAndWaitTillReady(cmd);
     }
   }
+  
+  /**
+   * Verifies the required pods are created, services are created and the servers are ready.
+   *
+   * @param maxIterations max iteration count
+   * @throws RuntimeException if pods/services of the domain are not created or WLS is not running
+   */
+  public void verifyDomainCreated(int maxIterations) throws Exception {
```

Comment:
don't duplicate logic like this.
make one method call the other.

Reply:
Refactored the code to remove the duplicate logic. This is the common area I'm testing comprehensively. Thanks for the comments.

Reply:
Done